context/design-tokens/README.md

```diff
@@ -2,156 +2,205 @@
 
 ## What Are Design Tokens
```

Comment:
Headings should be sentence case, and I suggest we use lower case, not title case, for design tokens throughout our docs. 
```suggestion
## What are design tokens?
```